src/model/formats/foreflight-reader.ts

```diff
@@ -80,31 +80,53 @@ export class ForeFlightReader {
   }
 
   private static checklistItemToEFIS(checklistItem: ForeFlightChecklistItem): ChecklistItem[] {
-    return [
+    const result = [
       checklistItem.type === ForeFlightUtils.ITEM_HEADER
-        ? ChecklistItem.create({
-            type: ChecklistItem_Type.ITEM_TITLE,
-            prompt: checklistItem.title,
-          })
-        : checklistItem.detail
-          ? ChecklistItem.create({
+        ? // Detail Item
```

Comment:
Style: these ternaries are getting pretty hard to follow, mind making them regular if/else control blocks?

Reply:
No problem, I've tried to improve the readability for you in 454463d - hopefully this makes it better.

I liked ternaries in this case because using a conditional expression instead of statements guarantees that you have considered all possible combinations of conditions. This is not synactically guaranteed with if-statements. But of course stronger correctness guarantees are of no use if the code is not understandable for humans :)